gwswlib/hydx.py

```diff
@@ -605,3 +625,11 @@ def check_headers(found, expected):
 
     if missing_columns:
         logger.error("missing columns found: %s", missing_columns)
+
+
+def isfloat(value):
```

Comment:
Qua naam: `is_float_value()`?
Bij de naam `isfloat()` dacht ik zelf gelijk aan `insinstance(3.14, float)`, maar jij bedoelt denk ik of de value (wat een string is) geconverteerd zou kunnen worden naar een float?